sdk/go/pulumi/config/config_test.go

```diff
@@ -46,12 +75,40 @@ func TestConfig(t *testing.T) {
 	assert.Equal(t, 42, cfg.GetInt("intint"))
 	assert.Equal(t, 99.963, cfg.GetFloat64("fpfpfp"))
 	assert.Equal(t, "", cfg.Get("missing"))
+	// missing key GetObj
+	err = cfg.GetObject("missing", &testStruct)
+	assert.Equal(t, emptyTestStruct, testStruct)
+	assert.Nil(t, err)
+	clearTestStruct(&testStruct)
```

Comment:
This is a bit non-idiomatic. Generally this would just be done inline:
```suggestion
	testStruct = TestStruct{}
```